### ISSUE TITLE
Cannot add properties to an empty Class Item

### ISSUE THREAD

**Issue by phd-hub** [nptr/msstyleEditor#37]
There is no option to add properties to Class Items that are empty.
e.g. in one instance I have:
```
Class: TaskBand
	Part: Common Properties
	Part: GROUPCONTENT
		State: Common
			Prop FONT (FONT) (503)
			Prop TEXTCOLOR (COLOR) (255, 255, 255)

and in another I have:

Class: TaskBand
	Part: Common Properties
		State: Common
			Prop BGTYPE (ENUM) (NONE)

	Part: GROUPCONTENT

```
and I cannot create the missing properties to the Class Parts that are empty.
Once a State exists then others can be added, but the Part itself cannot be added.
I therefore am unable to make one match the other.

**Comment by nptr** [comment]
It should be better now, see [pre-release V 1.4.0.5](https://github.com/nptr/msstyleEditor/releases/tag/1.4.0.5)

The "Common Properties" part now always has the "Common" state and is editable.

**Comment by phd-hub** [comment]
By default all other parts should also have the "Common", else no new properties can be added to those either (as shown in the GROUPCONTENT part in my original example). 
Maybe it just needs an option in the Properties window to "Add Common State" so it can be added as an when required.

**Comment by nptr** [comment]
Right...i forgot the "Common" state on parts where i don't know any states of :S

**Comment by nptr** [comment]
Here you go, i fixed it: [V 1.4.0.6](https://github.com/nptr/msstyleEditor/releases/tag/1.4.0.6)
I hope its good now.
